st2common/st2common/util/pack.py

```diff
@@ -41,6 +42,10 @@
     'normalize_pack_version'
 ]
 
+# Common format for python 2.7 warning
+PACK_PYTHON2_WARNING = "DEPRECATION WARNING: Pack %s only supports Python 2.x. " \
+                       "ST2 will remove support for Python 2.x in a future release."
```

Comment:
Small note about "future release(s)" and also would be nice to add a call to action for the users:

```suggestion
PACK_PYTHON2_WARNING = "DEPRECATION WARNING: Pack %s only supports Python 2.x. " \
                       "Python 2 support will be dropped in the future releases. " \
                       "Please consider updating your packs to work with Python 3."
```

Feel free to adjust as you wish.